### ISSUE TITLE
notice about empty share

### ISSUE THREAD

**Issue by konewa** [Vorski-Imagineering/share#4]

**Comment by rafallo** [comment]
Who should be noticed?
It means that we want to notify user that he didn't take a photo or fill a text field?

**Comment by vvorski** [comment]
@rafallo  yes, if no text and no photo was added then do not allow creating share. How do we notify of errors in other situations? If we do not have a standard way yet then we need to discuss design with @konewa and Kamil

**Comment by konewa** [comment]
@rimbambala  part of " layout on create Share❣ dialog"

**Comment by rimbambala** [comment]
It may be simple. We have already designed modal window for filter. A notice might look much the same. We just need to put there appropriate message.

**Comment by vvorski** [comment]
@konewa I leave it in your competent hands.

**Comment by vvorski** [comment]
Depends on style guide (error message display)

**Comment by vvorski** [comment]
Question is how do we display this error message... instead of pop-up display inline somehow on the create dialog?

**Comment by rimbambala** [comment]
![share-newshare-step2-12-error](https://cloud.githubusercontent.com/assets/6513797/2642557/f50a35bc-bf0a-11e3-9e24-0216e848d1f7.jpg)

**Comment by vvorski** [comment]
The error is "must include text or photo" so should we have the exclamation point over the photo as well?

**Comment by rimbambala** [comment]
Error message will have to be placed in the code - C3PO i think. Theres is no need for the design change. The exclamation mark can also be easily placed over photo. Upper-right corner

**Comment by vvorski** [comment]
@rimbambala yes, the point is that the user must include one of text or photo so where should the exclamation mark be?

Also is dark grey the best colour for an error message string background?

**Comment by rimbambala** [comment]
![share-newshare-step2-12-error](https://cloud.githubusercontent.com/assets/6513797/2653913/4596ed80-bfca-11e3-8625-3dc29533dd15.jpg)
Here's the position of the error. Dark gray is good. It stands out in comparison to other light colors so it will be clear and visible.

**Comment by vvorski** [comment]
something like this would be better:

![pulse_2](https://cloud.githubusercontent.com/assets/1789426/2657155/6fdb18f8-bff9-11e3-9fc0-66e1ed5ca2df.png)

**Comment by rimbambala** [comment]
![share-newshare-step2-12b-error](https://cloud.githubusercontent.com/assets/6513797/2667316/6370519c-c0ac-11e3-8ee5-ab3724d18bc1.jpg)
![share-newshare-step2-12a-error](https://cloud.githubusercontent.com/assets/6513797/2667317/63e3562e-c0ac-11e3-8d99-4fa75e5cc4d3.jpg)

**Comment by vvorski** [comment]
@rimbambala  Almost perfect - like the pulse tutorial example can we make the lines curved and pointing at the areas to click - I think that will make it clearer.

![share-newshare-step2-12-error](https://cloud.githubusercontent.com/assets/1789426/2700157/d8b3b9ba-c40f-11e3-97dc-f0016bd7ab83.jpg)

**Comment by rimbambala** [comment]
![share-newshare-step2-13-error](https://cloud.githubusercontent.com/assets/6513797/2704970/0ad14e28-c477-11e3-9118-296deb020b2f.jpg)

**Comment by vvorski** [comment]
I like it! Let's implement it...

**Comment by vvorski** [comment]
@rafallo  could you do this as well before monday? we need it for the launch...
